services/distribution/src/main/java/app/coronawarn/server/services/distribution/runner/Assembly.java

```diff
@@ -69,11 +73,14 @@ public void run(ApplicationArguments args) {
       outputDirectory.prepare(new ImmutableStack<>());
       logger.debug("Writing files...");
       outputDirectory.write();
+      logger.debug("Creating index file...");
+      masterIndexFileCreator.createIndex();
```

Comment:
Should happen as part of `prepare` and not as a separate step on the directory structure.